frontend/tests/e2e/features/addStudent.feature

```diff
@@ -0,0 +1,35 @@
+Feature: Register a new student
+    As a user
+    I want to register new students
+
+    Background:
+        Given the user has browsed the home page
+
+    Scenario: Successful student registration with no ielts score
+        When the administrator registers the student with following details without IELTS score
```

Comment:
```suggestion
        When the administrator registers the student with the following details
```
Maybe this can be a generic step for both `with` and `without` IELTS scores but the data in the data table might vary according to the requirement of the scenario. Can we do it like that?